cumulusci/tasks/release_notes/parser.py

```diff
@@ -295,16 +295,57 @@ def parse(self, change_note):
+        trial_info = self.release_notes_generator.trial_info
+        result = [self._render_header()]
+
         if version_id:
             version_id = urllib.parse.quote_plus(version_id)
-            return "\r\n".join(
-                [
-                    self._render_header(),
-                    "Production & Developer Edition Orgs:",
-                    f"{PROD_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
-                    "",
+            if (
+                self.release_notes_generator.sandbox_date
+                or self.release_notes_generator.production_date
+            ):
+                result += ["## Push Schedule", ""]
+                if self.release_notes_generator.sandbox_date:
+                    result += [
+                        f"Sandbox & Scratch Orgs: {self.release_notes_generator.sandbox_date}",
+                        f"{SANDBOX_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
+                    ]
+                    if self.release_notes_generator.production_date:
+                        result.append("")  # For formatting purposes
+                if self.release_notes_generator.production_date:
+                    result += [
+                        f"Production & Developer Edition Orgs: {self.release_notes_generator.production_date}",
+                        f"{PROD_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
+                    ]
+            else:
+                result += [
                     "Sandbox & Scratch Orgs:",
                     f"{SANDBOX_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
+                    "",
+                    "Production & Developer Edition Orgs:",
+                    f"{PROD_LOGIN_URL}/packaging/installPackage.apexp?p0={version_id}",
                 ]
-            )
-        return existing_content
+            if trial_info is True:
+                result += ["", "## Trialforce Template ID", "`TBD`"]
+            return "\r\n".join(result)
+        elif (
+            self.release_notes_generator.sandbox_date
+            or self.release_notes_generator.production_date
+        ):
+            result += ["## Push Schedule"]
+            if self.release_notes_generator.sandbox_date:
+                result.append(
+                    f"Sandbox orgs: {self.release_notes_generator.sandbox_date}"
+                )
+            if self.release_notes_generator.production_date:
+                result.append(
+                    f"Production orgs: {self.release_notes_generator.production_date}",
+                )
+            if trial_info is True:
+                result += ["", "## Trialforce Template ID", "`TBD`"]
+            return "\r\n".join(result)
+        elif trial_info is True:
+            result += ["## Trialforce Template ID", "`TBD`"]
+            return "\r\n".join(result)
+        else:
+            return existing_content
```

Comment:
I think you're making this more complicated than it needs to be by adding the links just below the push dates when both are available. I don't think that really makes sense (orgs which are getting the push will not be using the links).

I would stick to the current formatting (NPSP's Getting Started page parses the release notes to get the dates, so it's easiest if we don't change the format) so I think the logic can be something like this (in pseudocode):

```
if we don't have any of the options:
    return existing_content
start result with the header "Installation Info"
if production_date or sandbox_date:
    add section titled "Push Schedule" with the specified dates
if version_id:
    add section titled "Installation Links" with the links
if trial_info:
    add section titled "Trialforce Template ID"
return result
```